micropub.php

```diff
@@ -84,7 +84,7 @@ public static function init() {
 		add_filter( 'webfinger_user_data', array( $cls, 'micropub_jrd_links' ) );
 
 		// Disable adding headers if local auth is set
-		if ( ! MICROPUB_LOCAL_AUTH ) {
+		if ( MICROPUB_LOCAL_AUTH  || ! class_exists( 'IndieAuth_Plugin' ) ) {
```

Comment:
huh, just noticed, you need to keep the `!` before `MICROPUB_LOCAL_AUTH`, right?

good reminder that we always need manual testing for changes that aren't unit tested. :P

Reply:
oh and should the `||` be `&&`?

Reply:
No, because it disables if the declaration is not 0. So the ! was wrong. And it should be || because if the Indieauth plugin is installed, then it should be disabled.

Reply:
hrm, that's disappointing. looks like this was added in #103, which we supposedly manually tested and confirmed that it worked. so maybe we need to manually test more or better, since it wasn't enough before? or (maybe better :P) we need to add unit tests?

Reply:
I did test it. But point taken.

Reply:
ok, i understand the logic now. thanks!

i'm a little more nervous now about being sure all this changed auth logic is right, enough that we'll probably want to do a final round of manual testing before the net release. and i'll probably want us to add unit tests the next time we make any nontrivial changes. but i'll happily merge now!

Reply:
I understand. I intend to see about finishing the Indieauth work and having that out to test against. Some of the improvements and tests I may port up to Micropub.